utils/parsing.py

```diff
@@ -3,10 +3,13 @@
 
 
 def ParseModifiers(modifier_st: str):
+    if modifier_st == '':
+        return modifier_st
     return f"{'+' if modifier_st[0] not in ['+', '-'] else ''}{modifier_st}"
 
 
 def opposed_roll(roll: d20.RollResult, dc: d20.RollResult):
+    # print(f"{roll} - {dc}")
```

Comment:
Just adding a comment that we should make a debug logging function or something.  
Don't know how to do that personally, but would be real helpful for anyone working on it and not compile in at runtime.

Reply:
Taken to Issue: #106